### PR TITLE
Add union to list of API Headers

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Update list of APIHeaders

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml [hunk 1]
```diff
@@ -113,7 +113,7 @@
                 else
                 {
                     var lineNumberClass = $"line-number";
-                    @if (!String.IsNullOrEmpty(Model.CodeLine.CrossLanguageDefinitionId) && PageModelHelpers.IsAPIStubHeader(Model.CodeLine.DisplayString))
+                    @if (!String.IsNullOrEmpty(Model.CodeLine.CrossLanguageDefinitionId))
                     {
                         lineNumberClass = $"{lineNumberClass} cl-line-no";                     }
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml:118]: nit: Fix the new line and closing bracket.

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs [hunk 2]
```diff
@@ -139,18 +139,17 @@ public async Task<IActionResult> OnGetAsync(string id, string revisionId = null)
 
             if (!String.IsNullOrEmpty(ReviewContent.ActiveAPIRevision.Files.First().CrossLanguagePackageId))
             {
-                var correspondingReviewId = await _apiRevisionsManager.GetReviewIdsOfLanguageCorrespondingReviewAsync(ReviewContent.ActiveAPIRevision.Files.First().CrossLanguagePackageId);
-                var correspondingReviews = await _reviewManager.GetReviewsAsync(reviewIds: correspondingReviewId.Where(_ => _ != id).ToList(), isClosed: false);
-                foreach (var review in correspondingReviews)
+                var correspondingAPIRevisions = await _apiRevisionsManager.GetLanguageCorrespondingAPIRevisionsAsync(ReviewContent.ActiveAPIRevision.Files.First().CrossLanguagePackageId);
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs:142]: Won't this fetch all API revisions with same package ID? What will happen once all new revisions for a given package is created with this ID? what's the logic to identify the exact revision from this list.

File: src/dotnet/APIView/APIViewWeb/Repositories/CosmosAPIRevisionsRepository.cs [hunk 3]
```diff
@@ -258,19 +258,19 @@ public async Task<IEnumerable<APIRevisionListItemModel>> GetAPIRevisionsAssigned
         /// </summary>
         /// <param name="crossLanguagePackageId"></param>
         /// <returns></returns>
-        public async Task<IEnumerable<string>> GetReviewIdsOfLanguageCorrespondingReviewAsync(string crossLanguagePackageId)
+        public async Task<IEnumerable<APIRevisionListItemModel>> GetLanguageCorrespondingAPIRevisionsAsync(string crossLanguagePackageId)
         {
-            var query = $"SELECT DISTINCT VALUE c.ReviewId FROM c WHERE ARRAY_LENGTH(c.Files) > 0 AND c.Files[0].CrossLanguagePackageId = '{crossLanguagePackageId}'";
+            var query = $"SELECT DISTINCT c.id, c.ReviewId FROM c WHERE ARRAY_LENGTH(c.Files) > 0 AND ARRAY_CONTAINS(c.Files, {{ \"CrossLanguagePackageId\": \"{crossLanguagePackageId}\" }}, true)";
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Repositories/CosmosAPIRevisionsRepository.cs:263]: is cross language package ID unique for each revision? I assume not and it's a unique package level ID. So this query will return more than one revisions in each language. right?

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs (hunk 2 above)
Comment (chidozieononiwu) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs:142]: Yes, I thought of this. We can let the user select the APIRevision or find a way to default, maybe the latest automatic.

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs:142]: Can't we filter the revisions using associated PR number on top of cross language package Id to find the exact revision generated from same PR?

**Pull request closed without merging**